src/main/resources/cmc/db/migration/V2019_08_07_1545__migrate_legal_rep_reference_number_seq.sql

```diff
@@ -0,0 +1,13 @@
+CREATE SEQUENCE claim_legal_rep_reference_number_seq MAXVALUE 999999 NO CYCLE;
+
+/**
+ * Returns a new reference number from a 000LR001...999LR999 range.
+ */
+CREATE FUNCTION next_legal_rep_reference_number() RETURNS TEXT AS $$
+SELECT
+  regexp_replace(
+    to_char(
+      nextval('claim_legal_rep_reference_number_seq'),
+      'FM000000'),
```

Comment:
This is just for my understanding - what is `FM000000`? what are we trying to do here? is this claim reference pattern?